README.md

```diff
@@ -96,7 +96,17 @@ FAR CRs are deleted by NHC after it sees the Node is healthy again.
 
 #### Example CR
 
-The FAR, `FenceAgentsRemediation`, CR is created by the admin and is used to trigger the fence-agent on a specific node. The CR includes an *agent* field for the fence-agent name, *sharedparameters* field with all the shared, not specific to a node, parameters, and *nodeparameters* field to specify the parameters for the fenced node.
+The FAR CR (CustomResource), created by the admin, is used to trigger the fence-agent on a specific node.
+
+The CR includes the following parameters:
+
+* `agent` - the fence-agent name
+* `sharedparameters` - common cluster nodes information for executing the fence agent
+* `nodeparameters` - specific cluster nodes information for executing the fence agent
+* `retrycount` - the number of times to retry the fence-agent in case of failure. Optional parameter, default is 5.
+* `retryinterval` - the interval between retries in seconds. Optional parameter, default is "5s".
+* `timeout` - the timeout for the fence-agent in seconds. Optional parameter, default is "60s".
```

Comment:
NIT: I am unsure if I would use the word `the` at the beginning of each parameter explanation. It is fine without it, but it is just a minor preference, so feel free to ignore my comment.

Reply:
I prefer with the article honestly, but you're right that both ways sound correct

Reply:
I changed my mind, there would be a mix of descriptions some with the article and some without. Let's make them all without